app/models/concerns/hud_reports/destinations.rb

```diff
@@ -66,10 +66,10 @@ module HudReports::Destinations
         ['TOTAL', leavers_clause],
         [
           'Total persons exiting to positive housing destinations',
-          a_t[:project_type].in([1, 2]).
+          a_t[:project_type].in([0, 1, 2]).
```

Comment:
@ttoomey tagging you on these because I believe this is right (I looked a bit at the spec to confirm) but wanted to confirm I'm not missing some reason ES Entry/Exit wasn't included here.

Reply:
@eanders thanks, I think you're right. It looks like I just copied the logic from the previous report without adding ES-EE. Sorry I missed that!